docs/LSPApi.md

```diff
@@ -381,11 +381,20 @@ public class MyLSPDocumentSymbolFeature extends LSPDocumentSymbolFeature {
-| boolean isEnabled(PsiFile file)                       | Returns `true` if the LSP feature is enabled for the given file and `false` otherwise.                                                                                                                                             | `true`                      |
-| boolean isSupported(PsiFile file)                     | Returns `true` if the LSP feature is supported for the given file and `false` otherwise. <br/>This supported state is called after starting the language server, which matches the file and user with the LSP server capabilities. | Check the server capability |
-| boolean isExistingFormatterOverrideable(PsiFile file) | Returns `true` if existing formatters are overrideable and `false` otherwise.                                                                                                                                                      | `false`                     |
+| API                                                                  | Description                                                                                                                                                                                                                        | Default Behaviour                                      |
+|----------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------|
+| boolean isEnabled(PsiFile file)                                      | Returns `true` if the LSP feature is enabled for the given file and `false` otherwise.                                                                                                                                             | `true`                                                 |
+| boolean isSupported(PsiFile file)                                    | Returns `true` if the LSP feature is supported for the given file and `false` otherwise. <br/>This supported state is called after starting the language server, which matches the file and user with the LSP server capabilities. | Check the server capability                            |
+| boolean isRangeFormattingSupported(PsiFile file)                     | Returns `true` if the range formatting is supported for the given file and `false` otherwise.                                                                                                                                      | Check the server capability                            |

```

Comment:
I noticed that `isRangeFormattingSupported()` was missing from the docs so I added it.